godot-core/src/lib.rs

```diff
@@ -151,22 +149,22 @@ pub mod private {
 }
 
 macro_rules! generate_gdextension_api_version {
-    (
-        $(
-            ($name:ident, $gdextension_api:ident) => {
-                $($version:literal, )*
-            }
-        ),* $(,)?
-    ) => {
-        $(
-            $(
-                #[cfg($gdextension_api = $version)]
-                #[allow(dead_code)]
-                const $name: &str = $version;
-            )*
-        )*
-    };
-}
+     (
+         $(
+             ($name:ident, $gdextension_api:ident) => {
+                 $($version:literal, )*
+             }
+         ),* $(,)?
+     ) => {
+         $(
+             $(
+                 #[cfg($gdextension_api = $version)]
+                 #[allow(dead_code)]
+                 const $name: &str = $version;
+             )*
+         )*
+     };
+ }
```

Comment:
Also this shouldn't be related to the things your changing (and neither correct formatting), please revert 🙂